submissions/tosinKolawole/HTML/index.html

```diff
@@ -0,0 +1,58 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset=@"UTF-8">
+    <title>Infinix Note series</title>
+</head>
+<body>
+      <header>
+            <h1><b>INFINIX NOTE 5</b></h1>
+            <h4>The most amazing phone yet in the Infinix note series.</h4>
+       </header>
+
+      <figure>
+        <img src="https://images-na.ssl-images-amazon.com/images/I/61JH1324jTL._SL1040_.jpg">
+        <img src="https://www.gizmochina.com/wp-content/uploads/2018/06/Infinix-Note-5-600x562.jpg">
+      </figure>
+
+     <!--This section is for product price-->
+      <section>
+          <h2><b>Product Price<b></h2>
+          <article>
+             <p> 
+              The Infinix Note 5 Price in Nigeria is best on Beta Market Online stores and its current price starts at N57,000 naira for the 3GB variety and N65,000 naira for the 4GB variety.    
+             </p>
+ 
```

Comment:
You did not close this section.

Reply:
This was the previous index.html file I made, it was supposed to be replaced with the new one. I think I have been able to delete it. Thank you